docs/edge.md

```diff
@@ -234,9 +234,8 @@ An example of its output:
 
 ### Update cached pairs with the latest data
 
-```bash
-freqtrade edge --refresh-pairs-cached
-```
+Edge requires historic data the same way than backtesting does.
```

Comment:
"the same way as"

Reply:
"Don't merge" to keep it till the new release?

Reply:
I'm not sure - we removed `--live` in the previous release, and deprecating this for the next release will allow us to remove this option faster (?)

Reply:
Well, since we do not have a build number in the -dev version, it may be quite complex to understand which particular version of freqtrade the user runs (especially with that your conda). So if we change user interface I guess it's better for a while to make those changes along with changing the version of both develop and master...

Is it possible to introduce the build number into the -dev versioning, which will change with every merge, as ccxt employs, for example?

Reply:
> especially with that your conda

I'm not sure what conda has to do with it ... using the conda environment file installs the bot using `pip install -e .` ... just as the setup-script does.

The only thing it does differently is using most dependencies precompiled from the anaconda cloud, which has advantages on certain platforms (windows, raspberry)...